addons/binding/org.openhab.binding.camera/src/main/java/org/openhab/binding/camera/internal/CameraHandlerFactory.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2014 openHAB UG (haftungsbeschraenkt) and others.
```

Comment:
It seems that this file has a missing, outdated or otherwise non standard copyright header, can you fix all of them by running 

`mvn license:format` 

Please commit only the ones for your binding.

Reply:
OK

Reply:
@martinvw When I run 'mvv license:format' in the openhab2-addons base dir it wants to download the whole eclipse history at extreme slow speed, what is it doing there and can I shorten this?

Reply:
OK, found a way. Headers now adjusted.